shared/rakefiles/xk_infra.rake

```diff
@@ -88,7 +88,14 @@ task :apply_common_infra => [@gcp_creds_file] do
     #{@exekube_cmd} gcloud organizations get-iam-policy #{ENV["ORGANIZATION_ID"]} --format=json
   }
   permissions_list = JSON.parse(permissions_list_json)["bindings"]
-  organizations_permissions = ["roles/resourcemanager.projectCreator", "roles/billing.user"]
+  organizations_permissions = ["roles/billing.projectManager",
+                               "roles/billing.user",
+                               "roles/iam.organizationRoleViewer",
+                               "roles/iam.serviceAccountAdmin",
+                               "roles/iam.serviceAccountKeyAdmin",
```

Comment:
Same as ^

Reply:
`rake destroy_sa_keys` requires `roles/iam.serviceAccountKeyAdmin` because it removes SA keys